submissions/bukkyOyetimehin/challenge3-arrays/question3.js

```diff
@@ -0,0 +1,52 @@
+function findMissingNo(arr, n) {
```

Comment:
This is nice. I think I understand your approach and I think it would work. However, I could not get it to work. The function returns NaN for `[1,2,0]` and even the array declared on line 46. 

Excellent Time and Space Complexities too.